src/controller/model/endpoint.ts

```diff
@@ -469,13 +453,22 @@ class Endpoint extends Entity {
         );
     }
 
+    public getBindIndex(clusterId: number, target: Endpoint | Group): number {
```

Comment:
Maybe we should an additional `hasBind()`? That will be more clear for clients.

Reply:
Done!